source/MaterialXRuntime/RtStage.cpp

```diff
@@ -35,6 +35,16 @@ void RtStage::removeReference(const RtToken& name)
     data()->asA<PrvStage>()->removeReference(name);
 }
 
+void RtStage::copyReferences(RtObject stage)
```

Comment:
With this you can copy over library references from 1 stage to another.

Reply:
If we add the general access methods above I think this very specific method could be removed.

Reply:
Will remove.